docs/docs/configuration.md

```diff
@@ -22,26 +22,26 @@ From the command line:
 $ redis-server --loadmodule ./redisgraph.so OPT1 VAL1
 ```
 
-## Passing configuration options at run-time
+## Setting run-time configuration parameters at run-time
```

Comment:
maybe
Setting run-time configuration parameters

Reply:
I phrased it this way intentionally. There were several issues where people tried to change load-time configuration parameters using GRAPH.CONFIG (and because you can also set run-time configuration parameters at load-time).